.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,6 +4,14 @@ about: Submit a bug report to help us improve Habitat
 
 ---
 
+## Habitat-Sim version
+vX or master?
```

Comment:
`vX` doesn't associate with version number maybe use `vx.x.x` or just drop it and ask to specify commit or version to the issue.